### ISSUE TITLE
Question about past covariates, input_chunk_length, output_chunk_length and using trained model in real-life problem.

### ISSUE THREAD

**Issue by thaokimctu** [unit8co/darts#2407]
Good day and thank you for this amazing library.
Currently I'm having some problem regarding to past_covariates, input_chunk_length and output_chunk_length. After tinkering with TCN model hyper-parameters, I settled on this config because it yields the most reasonable result:
model = TCNModel(
    **input_chunk_length=7**,
    **output_chunk_length=1**,
    n_epochs=100,
    dropout=0.1,
    dilation_base=2,
    weight_norm=True,
    kernel_size=5,
    num_filters=8,
    nr_epochs_val_period=1,
    random_state=0,
    save_checkpoints=True,
    model_name=model_name,
    force_reset=True,
    **generate_torch_kwargs()
)
The result of using model.predict(n=182) (my goal is to predict 6 months in the future):
![image](https://github.com/unit8co/darts/assets/111738694/6d9f6c03-ac4d-407e-9b7d-909993614a2a)
But after some research, I learned that by setting n>output_chunk_length, the model would start using the future values of past_covariates to make the predicts and I was only lucky because I split the data after making past_covariates. I don't want to change the output_chunk_length value because anything other than 1 would worsen the test result. 
In real-life, most of the columns in past_covariates wouldn't be known beforehand at the time of predictions so my question is: Do I also have to make predictions to all the columns in past_covariates in order to predict my target, or I just did the whole thing wrong?

**Comment by dennisbader** [comment]
Hi @thaokimctu, yes, for `n > output_chunk_length`, the model will access future values of your past covariates.
We show a warning in this case, to make users aware that this is happening. We explain the reason for this [here](https://github.com/unit8co/darts/issues/1822#issuecomment-1725458419).

You always need to think of "if I generated a forecast right now, what data do I have available and how far does it extend into the past and future".

There are only three ways I could think of to achieve this:
- increase the `output_chunk_length` to 182 (if you're not interested in the steps 1-181, you can also check out the `output_chunk_shift` model creation parameter)
- forecast the past covariates
- remove the past covariates

What might help?
- increase the input chunk length (look back window) as well. Your current  7-step look back for a 182 step forecast might be too short
- try other models as well (TCN is not always a top runner, `TiDEModel` usually performs pretty well on most datasets, try simpler models, regression models, ...)

**Comment by thaokimctu** [comment]
@dennisbader Sorry for the late reply, I will try out your suggestions. Can I ask directly under this issues in case of I have further questions?
